### PR TITLE
Add script to update ora2 workflows with truncated course_ids

### PR DESCRIPTION
**Pull request by kluo**
@stvstnfrd @caesar2164 @dcadams 

https://github.com/Stanford-Online/edx-ora2/pull/25

### CODE REVIEW

File: scripts/update_ora2_truncated_course_ids.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    training_workflows = StudentTrainingWorkflow.objects.annotate(course_id_len=Length('course_id')).filter(course_id_len=40)
+
+    full_course_ids = {}  # Keep local dict to avoid repeated database hits for the same course_id
+    for workflow in chain(peer_workflows, staff_workflows, training_workflows):
+        truncated_course = workflow.course_id
+        if truncated_course not in full_course_ids:
+            # Get full course_id from AssessmentWorkflow table
+            try:
+                assessment_workflow = AssessmentWorkflow.objects.filter(course_id__startswith=truncated_course)[:1].get()
+                full_course_ids[truncated_course] = assessment_workflow.course_id
```
Comment (caesar2164) [inline scripts/update_ora2_truncated_course_ids.py]: What happens if more than one course matches?

File: scripts/update_ora2_truncated_course_ids.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+"""
+from itertools import chain
+import os
+import django
+from django.db.models.functions import Length
+from openedx.core.djangoapps.monkey_patch import django_db_models_options
+
+
+def main():
+    from openassessment.assessment.models import PeerWorkflow, StaffWorkflow, StudentTrainingWorkflow
```
Comment (stvstnfrd) [inline scripts/update_ora2_truncated_course_ids.py]: Why aren't these top-level imports?

File: scripts/update_ora2_truncated_course_ids.py (hunk 1 above)
Comment (stvstnfrd) [reply scripts/update_ora2_truncated_course_ids.py]: Yeah, it does seem unlikely, but perhaps it should log and NOOP if `length != 1`?

**Comment by caesar2164** [comment]
👍

File: scripts/update_ora2_truncated_course_ids.py (hunk 2 above)
Comment (kluo) [reply scripts/update_ora2_truncated_course_ids.py]: Needs the django setup code in the if block

**Pull request merged**